tests/graph/test_osm_network_wrapper.py

```diff
@@ -0,0 +1,222 @@
+        _osm_network = OsmNetworkWrapper(config=_config_fixture, graph_crs=None)
+        _polygon = LineString([[0, 0], [1, 0], [1, 1]])
+        _link_type = ""
+        _network_type = ""
+        with pytest.raises(TypeError) as exc_err:
+            _osm_network.get_graph_from_osm_download(_polygon, _link_type, _network_type)
+
+        assert str(
+            exc_err.value) == "Geometry must be a shapely Polygon or MultiPolygon. If you requested graph from place name, make sure your query resolves to a Polygon or MultiPolygon, and not some other geometry, like a Point. See OSMnx documentation for details."
+
+    def test_get_graph_from_osm_download_with_invalid_network_type_arg(self, _config_fixture: dict):
+        _osm_network = OsmNetworkWrapper(config=_config_fixture, graph_crs=None)
+        _polygon = Polygon([(0., 0.), (0., 1.), (1., 1.), (1., 0.), (0., 0.)])
+        _link_type = ""
+        _network_type = ""
+        with pytest.raises(ValueError) as exc_err:
+            _osm_network.get_graph_from_osm_download(_polygon, _link_type, _network_type)
+
+        assert str(exc_err.value) == 'Unrecognized network_type ""'
+
+    @pytest.fixture
+    def _valid_network_polygon_fixture(self) -> BaseGeometry:
+        _test_input_directory = test_data.joinpath("graph", "test_osm_network_wrapper")
+        _polygon_file = _test_input_directory.joinpath("_test_polygon.geojson")
+        assert _polygon_file.exists()
+        _polygon_dict = nut.read_geojson(_polygon_file)
+        yield nut.geojson_to_shp(_polygon_dict)
+
+    @slow_test
+    def test_get_graph_from_osm_download_output(self, _config_fixture: dict,
+                                                _valid_network_polygon_fixture: BaseGeometry):
+        # 1. Define test data.
+        _osm_network = OsmNetworkWrapper(config=_config_fixture, graph_crs=None)
+        _link_type = ""
+        _network_type = "drive"
+
+        # 2. Run test.
+        graph_complex = _osm_network.get_graph_from_osm_download(
+            polygon=_valid_network_polygon_fixture,
+            network_type=_network_type,
+            link_type=_link_type
+        )
+        _osm_network.drop_duplicates(graph_complex)
+
+        # 3. Verify expectations
+        # reference: https://www.openstreetmap.org/node/1402598729#map=17/51.98816/4.39126&layers=T
+        _osm_id_node_to_validate = 4987298323
+        # reference: https://www.openstreetmap.org/way/334316041#map=19/51.98945/4.39166&layers=T
+        _osm_id_edge_to_validate = 334316041
+
+        assert isinstance(graph_complex, Graph)
+        assert _osm_id_node_to_validate in list(graph_complex.nodes.keys())
+        assert _osm_id_edge_to_validate in list(map(lambda x: x["osmid"], graph_complex.edges.values()))
+
+    @pytest.mark.parametrize("config", [
+        pytest.param({
+            "static": Path(__file__).parent / "test_data" / "graph" / "test_osm_network_wrapper" / "static",
```

Comment:
We have the test data directory reachable by the `test_data` variable in `tests`:
```python
from tests import test_data
```
```suggestion
            "static": test_data.joinpath( "graph", "test_osm_network_wrapper", "static"),
```